packages/doc/content/components/components/divider/index.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: 'Divider'
+metaTitle: 'Divider'
+metaDescription: 'Divider Component'
+---
+
+
+import DividerWeb from './divider-web.mdx';
+import DividerNative from './divider-native.mdx';
+
+
+# Divider
+
+
+<TabbedView>
+  <Tab title="Web">
+    <DividerWeb />
+  </Tab>
+
+  <Tab title="Native">
+    <DividerNative />
+  </Tab>
+</TabbedView>
+
+  
```

Comment:
I believe that this documentation in general is too weak right now. As an effort to ensure engineers can rely more on documentation we're bringing new standards for this.

While it's not explicit defined, we can use references such as [Spectrum's divider](https://opensource.adobe.com/spectrum-web-components/components/divider/). It is a simple component but I think this may make the difference to someone contributing either from inside or outside Gympass, what are your thoughts on this?